### ISSUE TITLE
Long term support?

### ISSUE THREAD

**Issue by jacobrillema** [dsuryd/dotNetify#96]
I really would like to jump in on this with production code but my concern is support or availability years from now. I see that dotvvm has paid support behind them. What are you thoughts of how this will be supported? Any feedback is appreciated and thanks for all your help on this project!

**Comment by dsuryd** [comment]
I'm in the opinion that the key to ensure longevity of any non-corporate-backed open source project is the ability for the project to either transition from a single maintainer to a community of maintainers with patron backing (like Vue, Webpack),  or develop an economic model that keeps the core free and selling value-added services to businesses (ala Sidekiq, Gatsby) to sustain and fund its growth.

As I much would love to, I cannot say that this will happen with this project, because it really depends on whether it can grow its adoption rate, and has strong advocacy from early adopters.  Thank you for expressing your interest, and I hope others can voice theirs as well and be advocates, so dotNetify can have a good future!  :)

**Comment by jacobrillema** [comment]
I really appreciate the feedback. I guess I am still a bit nervous on jumping in but based on your honest feedback and all your hard work I think it is worth a try. I did some looking at your code and it is very clean and you do run the project very well here at GitHub. 

At least I could start out the MVP of our project and field test the feasibility. Part of my hesitation is being burned in the past with Silverlight and this also would be my first big jump into using open source frameworks. 

Again thanks for all your feedback. BTW, I didn't see in any documentation but is there a way we could fork the repo and build locally? I like to start from the build and work my way back on understanding the code.

**Comment by dsuryd** [comment]
Yes, Github provides good capability to fork projects and to merge changes (pull requests).  See their documentation here: https://help.github.com/.   Visual Studio and VS Code also has extensions to do it through the IDE.

**Comment by jacobrillema** [comment]
Is there any setup steps required for building locally? Like having Node installed? Do we just build the solution/csproj files? Do you use webpack at all?

**Comment by dsuryd** [comment]
The ASP.NET Core projects use Webpack.  You need to install Node and .NET SDK 2.1 (should be there if you install ASP.NET Core 2.1), and then just build the solution with latest VS 2017.

**Comment by jacobrillema** [comment]
Cool, thanks for the feedback! BTW did we get a Typescript version of the React template available yet to setup a new project from?

**Comment by dsuryd** [comment]
I didn't publish it to nuget but it's here courtesy of @MrFoxPro : https://github.com/dsuryd/dotnetify-react-demo-vs2017/tree/master/ReactTemplateTS

**Comment by jacobrillema** [comment]
Thanks I will give it a try.
